config/samples/backends/lvm-iscsi/backend.yaml

```diff
@@ -0,0 +1,26 @@
+# This sample requires an LVM VG with the name of `cinder-volumes` to exist on the OpenShift node where cinder-volume runs
+# For CRC this can be achieved with the `hack/dev/crc-create-lvm.sh`
+#
+# It also requires that iscsid and multpathd are running on the host.
+# The correct way to configure and run these daemons is using `MachineConfig` as shown in `iscsid.yaml` and `multipathd.yaml`.
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        lvm-iscsi:
+          customServiceConfig: |
+            [lvm]
+            image_volume_cache_enabled=false
+            use_multipath_for_image_xfer=true
```

Comment:
I have some enhancements to the default cinder.conf in mind, and would add this to the [backend_defaults] section. My understanding is there's no reason to _not_ set it True, so might as well make it the default setting for every backend that might use it.